src/telemetry.js

```diff
@@ -10,7 +10,8 @@ function Telemeter(options) {
 }
 
 Telemeter.prototype.configure = function(options) {
-  this.options = _.extend(true, {}, options);
+  var oldOptions = this.options;
+  this.options = _.extend(true, {}, oldOptions, options);
```

Comment:
I just copied how the notifier does it: https://github.com/rollbar/rollbar.js/blob/b6a3fce240bee0be185cf8e9982f321589bba8dc/src/notifier.js#L26-L27